services/src/unit-test/java/uk/gov/moj/sdt/services/config/ServicesConfigTest.java

```diff
@@ -0,0 +1,101 @@
+package uk.gov.moj.sdt.services.config;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.Spy;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import org.springframework.jms.listener.DefaultMessageListenerContainer;
+import org.springframework.jms.listener.adapter.MessageListenerAdapter;
+import org.springframework.transaction.PlatformTransactionManager;
+
+import javax.jms.ConnectionFactory;
+import java.util.HashMap;
+import java.util.Map;
+
+import uk.gov.moj.sdt.services.messaging.QueueConfig;
+import uk.gov.moj.sdt.services.utils.GenericXmlParser;
+import uk.gov.moj.sdt.services.utils.IndividualRequestsXmlParser;
+
+@ExtendWith(MockitoExtension.class)
+class ServicesConfigTest {
+
+@Mock
+    private ConnectionFactory mockConnectionFactory;
+@Mock
+    private PlatformTransactionManager mockPlatformTransactionManager;
+@Spy
+    private QueueConfig mockQueueConfig;
+@Mock
+    private MessageListenerAdapter mockMessageListenerAdaptor;
+
+    private DefaultMessageListenerContainer defaultMessageListenerContainer;
+
+    @Test
+    void testMessageListenerContainer(){
+
+        ServicesConfig mockServicesConfig = Mockito.spy(ServicesConfig.class);
```

Comment:
Any reason you're using spy to create the ServicesConfig object?  The test seems to work OK if you just create one normally as the other tests in the class do.

Reply:
think this can be set to resolved